backend/views/home.pug

```diff
@@ -77,3 +86,19 @@ block content
 					h6 Ming Sheng Quah
```

Comment:
Is Ming Shen Quah meant to be there?

Reply:
Pretty sure? That's how it was on the original

Reply:
Sorry meant all the above team members as well, wasn't sure if we were keeping them.

Reply:
Will leave it for now, I'll go ahead and merge